backend/src/interfaces/Post.ts

```diff
@@ -4,11 +4,11 @@ export interface Post {
     id: string;
     title: string;
     votes: number;
-    author: User;
-    lastVoted: Votes[];
+    author: string;
```

Comment:
ahh you changed the type from typte user to type string. seems to be easier right?